packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -2879,3 +2880,48 @@ test('manager notifiers', async t => {
     totalDebt: { value: 0n },
   });
 });
+
+test('governance publisher', async t => {
+  const { aeth } = t.context;
+  t.context.loanTiming = {
+    chargingPeriod: 2n,
+    recordingPeriod: 10n,
+  };
+
+  const services = await setupServices(
+    t,
+    [500n, 15n],
+    aeth.make(900n),
+    undefined,
+    undefined,
+    500n,
+  );
+  const { lender } = services.vaultFactory;
+  const directorGovNotifier = makeNotifierFromAsyncIterable(
+    E(lender).getElectorateSubscription(),
+  );
+  let {
+    value: { current },
+  } = await directorGovNotifier.getUpdateSince();
+  // can't deepEqual because of non-literal objects
+  t.is(current.Electorate.type, 'invitation');
+  t.is(current.LiquidationInstall.type, 'installation');
+  t.is(current.LiquidationTerms.type, 'unknown');
+  t.is(current.MinInitialDebt.type, 'amount');
+  t.is(current.ShortfallInvitation.type, 'invitation');
```

Comment:
`ShortfallInvitation` sounds like something more powerful than we usually publish. I should figure out what it is/does, but @Chris-Hibbert maybe you know offhand?

Reply:
If the governed value is an invitation, the published value is the invitation amount, which is not powerful.